src/dune_analytics.py

```diff
@@ -241,7 +241,7 @@ def execute_query(self):
         }
         self.handle_dune_request(query_data)
 
-    def query_result_id(self):
+    def query_result_id(self):  # type: ignore
```

Comment:
I think we can use int here, don't we?

Reply:
I wasn't sure if its int or string. I will try with int, but if that doesn't work then I will do this later! Thanks for reminding me.